web-server/v0.4/e2etests

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-cd $(dirname $0)
+# FIXME - Why do we have to CD to the directory where this script is
```

Comment:
We need to be able to reference `package.json` when invoking yarn commands in this script.

Reply:
Updated the comment removing the `FIXME` and replacing it with the reason.